include/behaviortree_cpp_v3/decorators/force_running_node.h

```diff
@@ -0,0 +1,65 @@
+/*  Copyright (C) 2018-2020 Davide Faconti, Eurecat -  All Rights Reserved
```

Comment:
I think you can add your name here Francisco :wink:

Reply:
Thanks!! I was not sure 😊